website/content/docs/commands/tls/cert-create.mdx

```diff
@@ -35,18 +35,16 @@ Usage: `nomad tls cert create [options]`
 - `-days=<int>`: Provide number of days the certificate is valid for from now
   on. Defaults to 1 year.
 
-- `-dc=<string>`: Provide the datacenter. Matters only for `-server`
-  certificates. Defaults to `dc1`.
+- `-cluster-region=<string>`: DEPRECATED please use `-region`.
```

Comment:
Let's add a note to [upgrade-specific.mdx](https://github.com/hashicorp/nomad/blob/main/website/content/docs/upgrade/upgrade-specific.mdx) for 1.6.0 that the `-cluster-region` flag of this command is deprecated and will be removed in Nomad 1.7.0